demo/src/App.vue

```diff
@@ -14,7 +15,7 @@
 import { onMounted } from 'vue';
 import resources from './assets/resources';
 import DemoPlugin from '@/DemoPlugin';
-import { ComponentDrawOption, FileInput } from 'leto-modelizer-plugin-core';
+import { Component, ComponentDrawOption, FileInput } from 'leto-modelizer-plugin-core';
```

Comment:
divide (3 or more)

Reply:
Done

Reply:
Not done, because is not a linter error.